.travis.yml

```diff
@@ -1,5 +1,8 @@
 services:
   - docker
+cache:
+  directories:
+    - vendor/bundle
```

Comment:
If the cache is not working as expected, can we remove the cache logic from this PR to another WIP PR?

Reply:
I do change the directory to `/srv/app/vendor/bundle` and test it. If it does not fix the problem I will remove the caching tasks.

Reply:
:crossed_fingers:

Reply:
It doe not work, I remove it from the PR.